File: CS/ComboBoxColumnBoundToEnum/Controllers/HomeController.cs
using ComboBoxColumnBoundToEnum.Models;
using DevExpress.Web.Mvc;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace ComboBoxColumnBoundToEnum.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            return View();
        }

        [ValidateInput(false)]
        public ActionResult GridViewPartial()
        {
            return PartialView("_GridViewPartial", OrderHelper.GetOrders());
        }
    }
}

File: VB/ComboColumnBoundToEnum/Controllers/HomeController.vb
Imports DevExpress.Web.Mvc
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        Return View()
    End Function

    <ValidateInput(False)>
    Public Function GridViewPartial() As ActionResult
        Return PartialView("_GridViewPartial", OrderHelper.GetOrders())
    End Function
End Class